oak/server/logging_channel.h

```diff
@@ -27,7 +27,8 @@ class LoggingChannelHalf final : public ChannelHalf {
   uint32_t Write(absl::Span<const char> data) override {
     std::string log_message(data.cbegin(), data.cend());
     LOG(INFO) << "LOG: " << log_message;
-  };
+    return data.size();
```

Comment:
Why did this work before? Isn't this a compile error?

Reply:
Only a compiler warning, apparently.  The return value isn't currently used, so we technically get away with the resulting undefined behaviour.

Reply:
Should we treat warnings as errors? I think, at least initially this might be better

Reply:
Yeah, that would make sense (as long as we can limit it to the code that's under our control).